docs/api/dialog.md

```diff
@@ -181,6 +191,11 @@ The `window` argument allows the dialog to attach itself to a parent window, mak
 The `filters` specifies an array of file types that can be displayed, see
 `dialog.showOpenDialog` for an example.
 
+**Note:** On Linux `defaultPath` is not supported when using portal file chooser
```

Comment:
`defaultPath` is only unsupported on versions <4 for open dialogs - save dialogs work as expected.

Reply:
Thanks for catching.

For reference, https://github.com/flatpak/xdg-desktop-portal/issues/796